### PR TITLE
[msal-angular] Update configuration.md with platformBrowserDynamic

### PR DESCRIPTION
**Pull request by jo-arroyo**
This PR updates the msal-angular-v2 configuration doc with details about how to configure msal-angular with platformBrowserDynamic.

### CODE REVIEW

**Review by jasonnutter — APPROVED** [review]

File: change/@azure-msal-angular-425d4dc5-5336-4bad-a3fe-f50a0137217e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (tnorling) [inline change/@azure-msal-angular-425d4dc5-5336-4bad-a3fe-f50a0137217e.json:2]:
```suggestion
  "type": "none",
```

File: lib/msal-angular/docs/v2-docs/configuration.md [hunk 2]
```diff
@@ -145,3 +146,108 @@ export function MSALGuardConfigFactory(): MsalGuardConfiguration {
 })
 export class AppModule { }
 ```
+
+## platformBrowserDynamic
+
+If you need to dynamically configure MSAL Angular (e.g. based on values returned from an API), you can use `platformBrowserDynamic`.
```
Comment (tnorling) [inline lib/msal-angular/docs/v2-docs/configuration.md:152]: Can you add a little explanation about what platformBrowserDynamic is and/or a link that explains it?

**Review by tnorling — APPROVED** [review]
One comment, looks good otherwise

File: lib/msal-angular/docs/v2-docs/configuration.md (hunk 2 above)
Comment (pkanher617) [reply lib/msal-angular/docs/v2-docs/configuration.md:152]: ^^

**Review by pkanher617 — APPROVED** [review]

File: lib/msal-angular/docs/v2-docs/configuration.md (hunk 2 above)
Comment (jo-arroyo) [reply lib/msal-angular/docs/v2-docs/configuration.md]: Done!

**Pull request merged**